diff --git a/binder/Makefile b/binder/Makefile
index b4ac7d8..f86e68f 100644
--- a/binder/Makefile
+++ b/binder/Makefile
@@ -1,4 +1,5 @@
-ccflags-y += -I$(src) -Wno-int-conversion -Wno-error=incompatible-pointer-types  -DCONFIG_ANDROID_BINDER_DEVICES="\"hostbinder,hostvndbinder,hosthwbinder\"" -DCONFIG_ANDROID_BINDERFS
+#ccflags-y += -I$(src) -Wno-int-conversion -Wno-error=incompatible-pointer-types  -DCONFIG_ANDROID_BINDER_DEVICES="\"hostbinder,hostvndbinder,hosthwbinder\"" -DCONFIG_ANDROID_BINDERFS
+ccflags-y += -I$(src) -Wno-int-conversion -Wno-error=incompatible-pointer-types -Wno-implicit-function-declaration -DCONFIG_ANDROID_BINDER_DEVICES="\"hostbinder,hostvndbinder,hosthwbinder\"" -DCONFIG_ANDROID_BINDERFS
 obj-m := binderfs_module.o binder_module.o
 binder_module-y := deps.o binder.o binder_alloc.o
 binderfs_module-y := deps.o binderfs.o
diff --git a/binder/binderfs.c b/binder/binderfs.c
index f54f842..22c3f1d 100644
--- a/binder/binderfs.c
+++ b/binder/binderfs.c
@@ -483,8 +483,10 @@ static struct dentry *binderfs_mount(struct file_system_type *fs_type,
 	if (!ns_capable(ipc_ns->user_ns, CAP_SYS_ADMIN))
 		return ERR_PTR(-EPERM);
 
-	sb = sget_userns(fs_type, binderfs_test_super, binderfs_set_super,
-			 flags, ipc_ns->user_ns, ipc_ns);
+	/*sb = sget_userns(fs_type, binderfs_test_super, binderfs_set_super,
+			 flags, ipc_ns->user_ns, ipc_ns);*/
+	sb = sget(fs_type, binderfs_test_super, binderfs_set_super,
+			 flags, ipc_ns);
 	if (IS_ERR(sb))
 		return ERR_CAST(sb);
 
